build-system-tests/templates/components/react/vite/template-tsconfig-vite-latest.json

```diff
@@ -20,12 +17,5 @@
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true
   },
-  "include": [
-    "src"
-  ],
-  "references": [
-    {
-      "path": "./tsconfig.node.json"
-    }
-  ]
```

Comment:
Remove the reference to `tsconfig.node.json` since that is now automatically being referenced in `tsconfig.json`